oshc/main/templates/contest_submission.html

```diff
@@ -4,7 +4,7 @@
 
 <div class="jumbotron text-xs-center">
   <div class="container text-center">
-      <h1 class="display-3">Thank You!</h1>
+      <h1 class="display-3 text-center">Thank you!</h1>
```

Comment:
This isn't working for me on my local or trial deployment. I have cache disabled. Can you please share screenshots?